helpers/postgres.rb

```diff
@@ -130,6 +130,50 @@ def generate_postgres_options(flavor: nil, location: nil)
     options.serialize
   end
 
+  PostgresLocation = Struct.new(:location, :available_postgres_versions, :available_vm_families)
```

Comment:
If instances don't need to be mutable:

```suggestion
  PostgresLocation = Data.define(:location, :available_postgres_versions, :available_vm_families)
```